src/mars.c

```diff
@@ -802,8 +801,29 @@ Language changes listed by -transition=id:\n\
             }
             else if (strcmp(p + 1, "ignore") == 0)
                 global.params.ignoreUnsupportedPragmas = true;
+
             else if (strcmp(p + 1, "property") == 0)
-                global.params.enforcePropertySyntax = true;
+            {
+                /*NOTE: -property used to disallow calling non-properties
+                without parentheses. This behaviour has fallen from grace.
+                Phobos dropped support for it while dmd still recognized it, so
+                that the switch has effectively not been supported. Time to
+                remove it from dmd.
+                Step 1 (2.068): Ignore -property and warn about it. */
+                warning(Loc(), "The -property switch has no effect since "
+                    "version 2.068.0. It's going to be deprecated and "
+                    "eventually removed in future versions.");
```

Comment:
You have to warn after the whole argument parsing is done b/c they way it's done now `-w -property` would warn while `-property -w` wouldn't, i.e. `global.params.warning` might still be false.